test/python/WMCore_t/Services_t/DBS_t/DBSUtils_t.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+"""
+_DBSUtils_t_
+
+Unit test for the DBSUtils module
+"""
+
+import time
+import unittest
+
+from WMCore.Services.DBS.DBSUtils import urlParams
+from WMCore.Services.DBS.DBS3Reader import DBS3Reader
+
+from nose.plugins.attrib import attr
+
+
+class DBSUtilsTest(unittest.TestCase):
+
+    def testUrlParams(self):
+        """
+        urlParams should return dictionary of URL parameters
+        """
+        url = 'http://a.b.com?d=1&f=bla'
+        results = urlParams(url)
+        self.assertCountEqual(results, {'d': '1', 'f': 'bla'})
+        self.assertTrue(results.get('d'), 1)
+        self.assertTrue(results.get('f'), 'bla')
+
+        url = 'http://a.b.com?d=1&f=bla&d=2'
+        results = urlParams(url)
+        self.assertCountEqual(results, {'d': ['1', '2'], 'f': 'bla'})
+        return
+
+
+    @attr("integration")
+    def testGetParallel(self):
+        """
+        test parallel execution of listDatasetFileDetails DBS API
+        We use small dataset with the following characteristics:
+
+		dasgoclient -query="dataset=/VBF1Parked/HIRun2013A-v1/RAW summary" | jq
+		[
+		  {
+			"file_size": 6053097,
+			"nblocks": 7,
+			"nevents": 0,
+			"nfiles": 7,
+			"nlumis": 428,
+			"num_block": 7,
+			"num_event": 0,
+			"num_file": 7,
+			"num_lumi": 428
+		  }
+		]
+
+		The parallel call should perform better then sequential while results
+        should remain the same.
+
+        """
+        url = "https://cmsweb.cern.ch/dbs/prod/global/DBSReader"
+        reader1 = DBS3Reader(url, logger=None, parallel=False, aggregate=True)
```

Comment:
Valentin, I like how you made this integration test. Can you please do the same for the other APIs that you offer a concurrent option (please keep them marked as integration, it's really only to make sure all these changes are providing the same result and to help testing whenever we need to change it again).